data/human/human missions.txt

```diff
@@ -5856,8 +5856,8 @@ mission "Quicksilver Mixup 2"
 		has "event: quicksilver mixup 2 available"
 	on offer
 		conversation
-			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
-			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
+			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a power tablet walks up to you after ending a hurried conversation with another pilot.`
```

Comment:
I'm think this is saying that the person "[power walks](https://dictionary.cambridge.org/dictionary/english/power-walk)" to you while carrying a tablet, not that they walk to you while carrying a "power tablet" (I don't know what that even would be).
```suggestion
			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
```

Reply:
Does "tablet power" make sense? It's quite hard to figure out what it means.

Reply:
I think in the context of the sentence it makes sense.

Reply:
Okay. Still not convinced though.